components/authz-service/projectassignment/projectassignment.go

```diff
@@ -0,0 +1,67 @@
+package projectassignment
+
```

Comment:
File naming convention for authz seems to be snake case, so should be project_assignment.go.